portable/GCC/ARM_CM0/port.c

```diff
@@ -34,6 +34,9 @@
 #include "FreeRTOS.h"
 #include "task.h"
 
+/* Prototype of all Interrupt Service Routines (ISRs). */
+typedef void ( * portISR_t )( void );
```

Comment:
In the replications within this PR, there is an accidental mix of the following styles:
`typedef void ( *portISR_t )( void );`
`typedef void ( * portISR_t )( void );`

And for additional context, here is code from timers.h with a third variation:
```
typedef void (* TimerCallbackFunction_t)( TimerHandle_t xTimer );

typedef void (* PendedFunction_t)( void * arg1,
                                   uint32_t arg2 );
```

Reply:
Thanks for catching Jeff. I fixed it.